packages/amplify-codegen/src/walkthrough/add.js

```diff
@@ -65,7 +66,7 @@ async function addWalkThrough(context, skip = [], withoutInit, decoupleFrontend,
   }
   if (!skip.includes('shouldGenerateDocs')) {
     answers.shouldGenerateDocs = await determineValue(inputParams, yesFlag, 'generateDocs', true, () => askShouldGenerateDocs());
-    answers.docsFilePath = getGraphQLDocPath(frontend, schemaLocation);
+    answers.docsFilePath = getGraphQLDocPath(frontend, docsFilePathDefault, answers.includePattern);
```

Comment:
Looks like we're using `includePattern` to determine the path to the generated docs. The walkthrough question that takes this info says `Enter the file name pattern of graphql queries, mutations and subscriptions`. As a developer, this sounds to me like I am being asked the path to existing docs if any. What is the purpose of `includePattern`? If it is to just used to store the generated docs, we might want to change it to `Enter the file name pattern for generated graphql queries, mutations and subscriptions`

Reply:
Yes this is a confusing part. The `includePattern` is an input param for type-gen. Originally CLI does not provide a question for the doc-gen output path. The current question is like assuming you have docs generated. However, from customers experience, I find that they tend to regard this question as the configure for doc-gen path. On the other hand, CLI does not give an option(question) to really change the `docFilePath` unless editting manually.

Reply:
From my perspective, the question you mention basically does two tasks:
1. What is your doc-gen output path
2. What is doc path as an input for type-gen
The reason for combining these two is that amplify codegen automatically combines these two process. There are few use cases that you only use type-gen alone. 

I would say this code changes are sufficient to solve users issues. However, you are right that we should not include those two tasks in only one question. I suggest that this can be done in the codegen redesign since adding/removing questions need additional API reviews/document changes.

Reply:
Thanks for the clarification. Yeah we need to separate these two questions and I agree that this will fix the current issues and we can take it up again at later point.